### ISSUE TITLE
Typing-friendly names

### ISSUE THREAD

**Issue by ajarifi** [ohno/Antique.jl#12]
I used the MorsePotential, but to call n_max is not easy. We have to copy it from the documentation.
Probably, it is useful to use typing-friendly names.

**Comment by ohno** [comment]
I agree with it is a little difficult to type. I'll think of something to do about it. For now, we can type `ₘ`  as `\_m` and tab key on REPL and also we can type it on VSCode by using [this plugin](https://marketplace.visualstudio.com/items?itemName=julialang.language-julia).

**Comment by ohno** [comment]
We will rename the following variables for https://github.com/ohno/Antique.jl/issues/97.
- `E` => `energy`
- `V` => `potential`
- `ψ` => `wavefunction`

**Comment by ohno** [comment]
Closing as resolved by #130.
